autofocus/predict/app/models/Predictor.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+
+from fastai.vision import load_learner, open_image
+
+
+MODEL_DIR = Path(__file__).resolve().parents[2] / "models"
+MODEL_NAME = "multilabel_model_20190407.pkl"
+model = load_learner(MODEL_DIR, MODEL_NAME)
+CLASSES = model.data.classes
+
+
+class Predictor:
```

Comment:
Could we make these methods top-level functions? I don't see much benefit in having them together in a class rather than just a module, it requires another line of code for callers, and it seems to me less idiomatic Python.

Reply:
Yes, I made them top-level functions now.
While developing the class I thought about adopting this class for other predictions as well. That was my thought behind this.